trino-aws-proxy/src/test/java/io/trino/aws/proxy/server/testing/harness/TrinoAwsProxyTestCommonModules.java

```diff
@@ -92,7 +92,7 @@ public TestingTrinoAwsProxyServer.Builder filter(TestingTrinoAwsProxyServer.Buil
         }
     }
 
-    public static class WithAllV4Containers
+    public static class WithStandardV4Containers
```

Comment:
`WithStandardContainersAndV4PySpark`? I was at first confused as to what `V4` referred to